src/app/components/ClientCard/OneTimeLinkBtn.vue

```diff
@@ -21,11 +23,38 @@ const _showOneTimeLink = useSubmit(
   {
     revert: async () => {
       await clientsStore.refresh();
+      await copyOneTimeLink();
     },
     noSuccessToast: true,
   }
 );
 
+async function copyOneTimeLink() {
+  if (!isSupported) {
+    toast.showToast({
+      type: 'error',
+      message: $t('copy.noSupport'),
+    });
+    return;
+  }
+  await nextTick();
+
+  await copy(formatOneTimeLink(document.location, props.client.oneTimeLink));
+
+  if (!copied.value) {
```

Comment:
when using an insecure connection copy does not work. this would show way to often for such cases